### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cgakey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,96 +1,24 @@
-# GitHub Gems: Driving Open-Source Investments With Data
+# Christopher Akey
```
Comment (edsioufi) [inline README.md]: I'd suggest keeping a title that relates to the project, like "Github Gems: Commit and Stars Data Pipeline" or anything else that resonates with you.

File: README.md [hunk 2]
```diff
@@ -1,96 +1,24 @@
...
-> ℹ️ Skip some steps if you're already set!
->
-> If you already have git, VSCode, and/or Python installed, just skip the corresponding step(s).
+What format would you like the data delivered in?
 
-1. [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+SQL database
 
-2. If you don't have a coding editor installed, [install VSCode](https://code.visualstudio.com/download). After that, [install the Python and Python extension](https://code.visualstudio.com/docs/languages/python#_install-python-and-the-python-extension).
+How do you currently analyze GitHub data?
```
Comment (edsioufi) [inline README.md]: Let's remove this last section as it relates to our fictional client (and only keep features that users of the repo would want to know about).

File: README.md [hunk 3]
```diff
@@ -1,96 +1,24 @@
-# GitHub Gems: Driving Open-Source Investments With Data
+# Christopher Akey
 
-Welcome to the GitHub Gems project! This project hosts a data analytics pipeline that enables smarter investment decisions by measuring the popularity of open-source repos on Github.
+What insights are you hoping to get?
```
Comment (edsioufi) [inline README.md:3]: All the content looks great! I'll encourage switching to titles that are more like real-life product documentation e.g. "Overview", "Metrics", etc.

**Review by edsioufi — COMMENTED** [review]
Great job! 🙌 

I've added suggestions to improve the readme a bit, as comments.

Also don't merge this PR before I review it as it's merging into a branch other than main (will explain when we chat).

**Comment by cgakey** [comment]
Commits were already on main

**Pull request closed without merging**